released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.2.0/opam

```diff
@@ -10,7 +10,7 @@ build: [ [ make "build"]
          [ make "test-suite" ] {with-test}
```

Comment:
```suggestion
```
I don't mind if 1.2.1 is released, but since the @gares confirms that 1.2.0 actually works with coq-elpi.1.2.0, how about just skipping the test suite as I suggested?

Reply:
Or we could even augment the `with-test` qualification with a version check...

Reply:
Ok, let's skip the test suite